cocos/game/game.ts

```diff
@@ -108,16 +108,18 @@ export interface IGameConfig {
      * - 0 - 通过引擎自动选择。
      * - 1 - 强制使用 canvas 渲染。
      * - 2 - 强制使用 WebGL 渲染，但是在部分 Android 浏览器中这个选项会被忽略。
-     * - 3 - 使用空渲染器，可以用于测试和服务器端环境，目前暂时用于 Cocos 内部测试使用
+     * - 3 - 强制使用 WebGPU 渲染，但是在部分浏览器中这个选项会被忽略
+     * - 4 - 使用空渲染器，可以用于测试和服务器端环境，目前暂时用于 Cocos 内部测试使用
```

Comment:
Shall we keep 3 as dummy renderer? This change might break compatibility. @minggo

Reply:
Yep, i think we should swap 3 and 4 to keep compatibility.